warehouse/deltalake/deltalake.go

```diff
@@ -548,6 +541,13 @@ func (dl *HandleT) loadTable(tableName string, tableSchemaInUpload warehouseutil
 	return
 }
 
+func loadTableHandler() LoadTable {
+	if loadTableStrategy == "APPEND" {
+		return &LTAppend{}
+	}
+	return &LTMerge{}
```

Comment:
If we are ever going to have two different strategy like append and merge and functionality is only limited to generating the sql statement different, should we represent them with simple conditional statement like
```
   if strategy == 'APPEND':
       return appendableSQLStatement()
   return mergeableSQLStatement()
```

Wdyt ?
I am saying this as this would prevent us to have unnecessary abstraction. Strategy pattern is mostly valid when you have many strategies and enough capabilities defined within a strategy.